posts/2022-12-14-android-ndk-update.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Updating the Android NDK in Rust <version>"
+author: Android Platform Team
+description: "Modernizing Android support in Rust"
+---
+
+We are pleased to announce that Android platform support in Rust will be
+modernized in Rust <version> as we update the target NDK from r17 to r25.  As a
+consequence the minimum supported API level will increase from 15 (Jelly Bean)
+to 19 (KitKat).
+
+In NDK r23 Android switched to using LLVMs `libunwind` for all architectures.
+This meant that if a project were to target NDK r23 or newer with previous
+versions of Rust a workaround would be required to redirect attempts to link
+against `libgcc` to instead link against `libunwind`.  Following this update
+this workaround will no longer be necessary.  Projects using NDKs between r17
```

Comment:
this should probably mention that the workaround will cause issues; I think people on https://github.com/rust-lang/rust/issues/103673 had more detailed suggestions.

Reply:
I've added language explaining the need to update for projects targeting an NDK between r17 and r23.